awscli/customizations/wizard/wizards/configure/main.yml

```diff
@@ -0,0 +1,361 @@
+version: "0.2"
+title: Configure the AWS CLI
+description: This wizard will create a new CLI config profile for you
+plan:
+  decide_profile_type:
+    shortname: Intro
+    values:
+      new_profile:
+        type: prompt
+        description: Create new profile or updated existing
+        choices:
+          - display: Create new
+            actual_value: yes
+          - display: Update existing
+            actual_value: no
+      new_profile_name:
+        type: prompt
+        description: Enter the name of the profile
+        condition:
+          variable: new_profile
+          equals: yes
+      existing_profiles:
+        type: sharedconfig
+        operation: ListProfiles
+      updated_profile_name:
+        type: prompt
+        description: Enter the name of the profile
+        choices: existing_profiles
+        condition:
+          variable: new_profile
+          equals: no
+      profile_name:
+        type: template
+        value: |
+          {% if {new_profile} == yes %}
+          {new_profile_name}{% endif %}
+          {% if {new_profile} == no %}
+          {updated_profile_name}{% endif %}
+  creds:
+    shortname: Credentials
+    values:
+      add_creds:
+        type: prompt
+        description: "Would you like to add credentials"
+        choices:
+          - display: Yes
+            actual_value: yes
+          - display: No
+            actual_value: no
+      config_type:
+        type: prompt
+        description: "Choose credentials type"
+        choices:
+          - display: Static Credentials
+            actual_value: static_creds
+          - display: Assume Role
+            actual_value: assume_role
+          - display: Process Provider
+            actual_value: proc_provider
```

Comment:
It would be really neat if there was a doc panel explaining what each of these credential providers do similar to the additional configuration prompts.